sycl/test-e2e/Regression/reduction_resource_leak_usm.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: level_zero, level_zero_dev_kit
-// TODO: ZE_DEBUG=4 produces no output on Windows. Enable when fixed.
 // XFAIL: windows
+// UNSUPPORTED: (ze_debug && windows)
```

Comment:
Could you elaborate more on this change, please?
What is the status of this test on Windows w/ and w/o `ze_debug` config option?

NOTE: I see that ZE_DEBUG is always set on line 6, so I'm not sure if `ze_debug` lit config option used correctly.

Reply:
> Should we disable it on Windows for now and open an internal JIRA for it?

Yes, open a tracker. Please leave original code with the comment and just replace `XFAIL` with `UNSUPPORTED`. 
@bso-intel, FYI.

Reply:
Done!